src/components/LocationPage/LocationPageHeading.tsx

```diff
@@ -10,10 +11,21 @@ const LocationPageHeading: React.FC<{ region: Region; isEmbed: boolean }> = ({
   } else if (region instanceof County) {
     return (
       <Fragment>
-        <strong>{region.name}, </strong>
-        <a href={`${isEmbed ? '/embed' : ''}/us/${region.state.urlSegment}`}>
-          {region.state.stateCode}
-        </a>
+        <strong>{region.name}</strong>
+        {', '}
+        <HeaderStateCode>
+          <a href={`${isEmbed ? '/embed' : ''}/us/${region.state.urlSegment}`}>
+            {region.state.stateCode}
+          </a>
+        </HeaderStateCode>
+      </Fragment>
+    );
+  } else if (region instanceof MetroArea) {
+    return (
+      <Fragment>
+        <strong>{region.name}</strong>
+        {', '}
+        <HeaderStateCode>{region.stateCodes}</HeaderStateCode>
```

Comment:
is state codes a string? Taking a look at the `MetroArea` object, I think the `stateCodes` + `stateCodesList` property might be unnecessary.  Joining them with a `-` seems a bit arbitrary and I think i would be more clear if `stateCodes` was a list of state codes and the logic on how to present them just lives here

Reply:
That makes sense, however, I think it makes sense to define a standard for rendering those so we don't have to join them each time and also to avoid having too many versions of the same thing (separated by comma, separated by dashes, etc).

Reply:
Definitely! Maybe even just formattedStateNames or something similar would be more clear. Definitely a nit and not a big deal though